core/src/validator.rs

```diff
@@ -1293,23 +1299,23 @@ fn wait_for_supermajority(
     cluster_info: &ClusterInfo,
     rpc_override_health_check: Arc<AtomicBool>,
     start_progress: &Arc<RwLock<ValidatorStartProgress>>,
-) -> bool {
+) -> (bool, bool) {
```

Comment:
hmmm could we add to the function comment on what the second boolean represents?

Reply:
Seems like this might be better modeled as a `Result<bool>` since we exit the validator whenever the first bool, which represents whether an error occurred, is `true`